mozdownload/scraper.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, directory, version, platform=None,
         self._target = None
         self._binary = None
 
-        self.directory = directory
+        self.destination = destination
```

Comment:
I think with this rename we have a duplication between `self.destination` and `self.target`. I assume we should get rid of `self.target`. Maybe we can make `self.destination` a write property, which contains the code as currently lives in `self.target`. I don't see why we would have to run the code more than once for the same session, especially when no input parameters are changing.